AlertaDengue/upload/models.py

```diff
@@ -0,0 +1,300 @@
+    MA = "MA", "Maranhão"  # pyright: ignore
+    MT = "MT", "Mato Grosso"  # pyright: ignore
+    MS = "MS", "Mato Grosso do Sul"  # pyright: ignore
+    MG = "MG", "Minas Gerais"  # pyright: ignore
+    PA = "PA", "Pará"  # pyright: ignore
+    PB = "PB", "Paraíba"  # pyright: ignore
+    PR = "PR", "Paraná"  # pyright: ignore
+    PE = "PE", "Pernambuco"  # pyright: ignore
+    PI = "PI", "Piauí"  # pyright: ignore
+    RJ = "RJ", "Rio de Janeiro"  # pyright: ignore
+    RN = "RN", "Rio Grande do Norte"  # pyright: ignore
+    RS = "RS", "Rio Grande do Sul"  # pyright: ignore
+    RO = "RO", "Rondônia"  # pyright: ignore
+    RR = "RR", "Roraima"  # pyright: ignore
+    SC = "SC", "Santa Catarina"  # pyright: ignore
+    SP = "SP", "São Paulo"  # pyright: ignore
+    SE = "SE", "Sergipe"  # pyright: ignore
+    TO = "TO", "Tocantins"  # pyright: ignore
+    DF = "DF", "Distrito Federal"  # pyright: ignore
+
+
+class Diseases(models.TextChoices):
+    DENGUE = "dengue", "Dengue"  # pyright: ignore
+    CHIK = "chik", "Chigungunya"  # pyright: ignore
+    ZIKA = "zika", "Zika"  # pyright: ignore
+
+
+class Status(models.TextChoices):
+    WAITING_CHUNK = "waiting_chunk", _("Aguardando chunk")  # pyright: ignore
+    CHUNKING = "chunking", _("Processando chunks")  # pyright: ignore
+    WAITING_INSERT = "waiting_insert", _(
+        "Aguardando inserção"
+    )  # pyright: ignore
+    INSERTING = "inserting", _("Inserindo dados")  # pyright: ignore
+    ERROR = "error", _("Erro")  # pyright: ignore
+    FINISHED = "finished", _("Finalizado")  # pyright: ignore
+
+
+class SINAN(models.Model):
+    db_engine = get_sqla_conn("dengue")
+    table_schema = '"Municipio"."Notificacao"'
+
+    filename = models.CharField(
+        primary_key=True,
+        null=False,
+        blank=False,
+        help_text=_("Name of the file with suffix"),
+        validators=[validade_file_type],
+        max_length=100
+    )
+    filepath = models.FileField(
+        null=True,
+        blank=False,
+        help_text=_("Absolute data file path, Null if deleted after insert"),
+        validators=[validate_file_exists]
+    )
+    disease = models.CharField(
+        choices=Diseases.choices,
+        default=Diseases.DENGUE,
+        max_length=50
+    )
+    notification_year = models.IntegerField(
+        null=False,
+        validators=[validade_year]
+    )
+    uf = models.CharField(
+        max_length=2,
+        null=False,
+        choices=UFs.choices,
+        default=UFs.BR
+    )
+    municipio = models.IntegerField(null=True)
+    status = models.TextField(
+        null=False,
+        choices=Status.choices,
+        default=Status.WAITING_CHUNK,
+        help_text=_("Upload status of the file")
+    )
+    parse_error = models.BooleanField(
+        null=False,
+        default=False,
+        help_text=_(
+            "An parse error ocurred when reading data, "
+            "moved errored rows to `error_residue` file. "
```

Comment:
change this line to:  "moved rows that failed parsing to `error_misparsed` file. "